size-plugin-browser.json

```diff
@@ -1 +1,408 @@
-[{"timestamp":1566495912208,"files":[{"filename":"ssr-build/editor.chunk.51cc7.css","previous":3174,"size":0,"diff":-3174},{"filename":"ssr-build/editor.chunk.907a2.js","previous":65855,"size":0,"diff":-65855},{"filename":"ssr-build/emoji.chunk.a7ea4.js","previous":8834,"size":0,"diff":-8834},{"filename":"ssr-build/markdown.e0970.worker.js","previous":8612,"size":0,"diff":-8612},{"filename":"ssr-build/prism.28075.worker.js","previous":6244,"size":0,"diff":-6244},{"filename":"ssr-build/repl.dde27.worker.js","previous":37054,"size":0,"diff":-37054},{"filename":"ssr-build/runner.chunk.82e44.js","previous":5856,"size":0,"diff":-5856},{"filename":"ssr-build/ssr-bundle.6d789.css","previous":10829,"size":0,"diff":-10829},{"filename":"ssr-build/ssr-bundle.js","previous":27557,"size":0,"diff":-27557},{"filename":"sw-esm.js","previous":0,"size":682,"diff":682},{"filename":"sw.js","previous":0,"size":678,"diff":678},{"filename":"markdown.bc303.worker.js","previous":0,"size":8681,"diff":8681},{"filename":"prism.18eb5.worker.js","previous":0,"size":6256,"diff":6256},{"filename":"repl.ed003.worker.js","previous":0,"size":35099,"diff":35099},{"filename":"bundle.51cd1.css","previous":0,"size":9071,"diff":9071},{"filename":"bundle.bd43d.esm.js","previous":0,"size":22857,"diff":22857},{"filename":"editor.chunk.51cc7.css","previous":0,"size":2154,"diff":2154},{"filename":"editor.chunk.d5373.esm.js","previous":0,"size":66307,"diff":66307},{"filename":"emoji.chunk.91398.esm.js","previous":0,"size":8854,"diff":8854},{"filename":"polyfills.6a525.esm.js","previous":0,"size":2128,"diff":2128},{"filename":"runner.chunk.7e9f0.esm.js","previous":0,"size":5789,"diff":5789},{"filename":"5.chunk.275db.css","previous":0,"size":1467,"diff":1467},{"filename":"5.chunk.d0b78.esm.js","previous":0,"size":3387,"diff":3387},{"filename":"markdown.8ff2c.worker.js","previous":0,"size":8682,"diff":8682},{"filename":"repl.80e6e.worker.js","previous":0,"size":36553,"diff":36553},{"filename":"bundle.137bc.js","previous":0,"size":23223,"diff":23223},{"filename":"editor.chunk.7148e.js","previous":0,"size":66352,"diff":66352},{"filename":"emoji.chunk.1ed91.js","previous":0,"size":8848,"diff":8848},{"filename":"polyfills.3233a.js","previous":0,"size":2124,"diff":2124},{"filename":"runner.chunk.ca897.js","previous":0,"size":5830,"diff":5830},{"filename":"5.chunk.50f21.js","previous":0,"size":3451,"diff":3451}]}]
+[
+  {
+    "timestamp": 1566971456944,
```

Comment:
@developit I am just trying to understand what causes the merge conflict.

<img width="1036" alt="Screen Shot 2019-08-28 at 9 38 40 PM" src="https://user-images.githubusercontent.com/10448534/63873131-a9c8ad80-c9dc-11e9-806b-a831cf096ca7.png">

looking at the timeline

- merging of #404 triggered #405 (#405 has changes corresponding to #404 )
- merging of #406 triggered #407 (#407 has changes corresponding to #406 )

Note: #407 was created before #405 was merged into master

As #405 was merged after #406 and both #405 & #407 were trying to add content on the same line number, so thus after merging of #405 into master we got conflicts in #407 🤦

maybe size-plugin-bot should simply close any the existing pr before opening a new one. this way we always have lastest `sizes` in pr

_I can be totally wrong here & this could be the worst theory that came into my mind_ 🙃